### PR TITLE
Add blog post for 2.15.0 release

### PR DESCRIPTION
**Pull request by yifanzou**
**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Flink/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Spark/lastCompletedBuild/)
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Spark_Batch/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python2/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python35/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python36/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python37/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_PVR_Flink_Cron/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Spark/lastCompletedBuild/)
XLang | --- | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_XVR_Flink/lastCompletedBuild/) | --- | --- | ---

Pre-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

--- |Java | Python | Go | Website
--- | --- | --- | --- | ---
Non-portable | [Build Status](https://builds.apache.org/job/beam_PreCommit_Java_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Go_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Website_Cron/lastCompletedBuild/) 
Portable | --- | [Build Status](https://builds.apache.org/job/beam_PreCommit_Portable_Python_Cron/lastCompletedBuild/) | --- | ---

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

### CODE REVIEW

File: website/src/_posts/2019-08-14-beam-2.15.0.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+limitations under the License.
+-->
+
+ We are happy to present the new 2.15.0 release of Beam. This release includes both improvements and new functionality.
+See the [download page]({{ site.baseurl }}/get-started/downloads/#2130-2019-08-14) for this release.<!--more-->
+For more information on changes in 2.15.0, check out the
+[detailed release notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12319527&version=12345489).
+
+ ## Highlights
+
```
Comment (iemejia) [inline website/src/_posts/2019-08-14-beam-2.15.0.md]: Probably good idea to add to the highlights the vendored dependency upgrades (but not 100% sure if worth)
- Vendored Guava was upgraded to version 26.0

Comment (yifanzou) [reply website/src/_posts/2019-08-14-beam-2.15.0.md]: Thanks for reviewing the blog! I added it to the highlight.

**Comment by yifanzou** [comment]
Run Website PreCommit

**Comment by yifanzou** [comment]
+R: @aaltay

File: website/src/_posts/2019-08-22-beam-2.15.0.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+* Support ParquetTable in SQL.
+* Add hot key detection to Dataflow Runner.
+* Enable throughput collection in BigQuery performance tests.
+
+
+ ### Bugfixes
+
+* Various bug fixes and performance improvements.
+
```
Comment (aaltay) [inline website/src/_posts/2019-08-22-beam-2.15.0.md]: Please add a known issue:

[BEAM-7616] urlopen calls may get stuck. (Not an issue in 2.14.0).

File: website/src/_posts/2019-08-22-beam-2.15.0.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+* Add AvroIO.sink for IndexedRecord (FileIO compatible).
+
+
+ ### New Features / Improvements
+
+* Support ParquetTable in SQL.
+* Add hot key detection to Dataflow Runner.
+* Enable throughput collection in BigQuery performance tests.
+
```
Comment (aaltay) [inline website/src/_posts/2019-08-22-beam-2.15.0.md]: Schema support added to JDBC sink [BEAM-6675]

File: website/src/_posts/2019-08-22-beam-2.15.0.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+ We are happy to present the new 2.15.0 release of Beam. This release includes both improvements and new functionality.
+See the [download page]({{ site.baseurl }}/get-started/downloads/#2130-2019-08-14) for this release.<!--more-->
+For more information on changes in 2.15.0, check out the
+[detailed release notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12319527&version=12345489).
+
+ ## Highlights
```
Comment (aaltay) [inline website/src/_posts/2019-08-22-beam-2.15.0.md]: Can you make a pass over the JIRA list and see what other interesting things are there?

Comment (yifanzou) [reply website/src/_posts/2019-08-22-beam-2.15.0.md]: The release note contains all JIRAs is listed above.

File: website/src/_posts/2019-08-22-beam-2.15.0.md (hunk 3 above)
Comment (yifanzou) [reply website/src/_posts/2019-08-22-beam-2.15.0.md]: done.

File: website/src/_posts/2019-08-22-beam-2.15.0.md (hunk 2 above)
Comment (yifanzou) [reply website/src/_posts/2019-08-22-beam-2.15.0.md]: done

**Pull request merged**